tic-tac-toe/src/app/components/game/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import Board from '../board';
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      history: [{ squares: Array(9).fill(null) }],
+      stepNumber: 0,
+      xIsNext: true
+    };
+  }
+
+  calculateWinner = squares => {
+    const lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
```

Comment:
You can create a constants file in the src folder and you can extract lines to this file.